flash/core/data/io/input_base.py

```diff
@@ -236,18 +236,6 @@ def register_input_transform(
             )
         cls.input_transforms_registry(fn=fn, name=enum)
 
-    @property
-    def dataloader_collate_fn(self) -> Optional[Callable]:
-        if self.transform:
-            self.transform.running_stage = self.running_stage
-            return self.transform.dataloader_collate_fn
-
-    @property
-    def on_after_batch_transfer_fn(self) -> Optional[Callable]:
-        if self.transform:
-            self.transform.running_stage = self.running_stage
-            return self.transform.on_after_batch_transfer_fn
-
```

Comment:
Not sure about this change. It makes sense to me that the input would contain everything related to the transforms as that is the standard contract of a pytorch dataset. We are just extending it to include a collate and a device transform.

Reply:
Added back !